docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-enable-cluster-label-verification/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+description: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+title: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+menuTitle: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
```

Comment:
Titles and headings should use sentence case rather than title case.
We should also avoid gerunds ("ing" endings in verbs).
Titles should also use the full product name "Grafana Mimir" for search engine optimization.

I have corrected this as an example, but please see my general review comment because we may want to rename this page.

```suggestion
title: "Migrate Grafana Mimir Helm chart to use memberlist cluster label"
```

Also the `menuTitle` can be omitted if it is the same as the title.
In this case, I think we need a much shorter title that omits the product name.
Considering a page rename may make this `menuTitle` much shorter, perhaps `Configure a memberlist cluster label`?